content/en/blog/_posts/2022-08-04-cgroupv2-ga.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog
+title: "CGroup V2 Goes GA in 1.25"
```

Comment:
```suggestion
title: "Kubernetes 1.25: cgroup v2 graduates to GA"
```
Or stable, IDK what the right terminology is

Suggestion is for consistency with other feature graduation blogs in the k8s blog

Reply:
Good idea, used GA for now.